README.md

```diff
@@ -40,15 +52,91 @@ or
+calendar = calendar.map_years(2020, 2021)
+calendar
+>>>    i_interval                 (target) 0                         1
+>>>    anchor_year
+>>>    2021         (2021-06-03, 2021-11-30]  (2020-12-05, 2021-06-03]
+>>>    2020         (2020-06-03, 2020-11-30]  (2019-12-06, 2020-06-03]
+```
+
+`s2spy` works with `xarray` data. Now, the user can load their data and easily resample their data to the desired timescales configured in the calendar:
+
+```py
+calendar = calendar.map_to_data(input_data)
+bins = s2spy.time.resample(calendar, input_data)
+>>>       anchor_year  i_interval                  interval  mean_data  target
+>>>     0        2020           0  (2020-06-03, 2020-11-30]      275.5    True
+>>>     1        2020           1  (2019-12-06, 2020-06-03]       95.5   False
+>>>     2        2021           0  (2021-06-03, 2021-11-30]      640.5    True
+>>>     3        2021           1  (2020-12-05, 2021-06-03]      460.5   False
+```
+
+Depending on their tasks, the user can choose their desired calendar from a collection of different types of calendars, to process their data (e.g. `MonthlyCalendar` and `WeeklyCalendar`).
+
+### Cross-validation
+Using `s2spy`, users can easily perform cross-validation by calling a splitter and generating train/test splits with the resampled data:
+
+```py
+bins = s2spy.time.resample(calendar, input_data)
+
+from sklearn.model_selection import ShuffleSplit
+splitter = ShuffleSplit(n_splits=3)
+s2spy.traintest.split_groups(splitter, bins)
+```
+
+All splitter classes from `scikit-learn` are supported, a list is available [here](https://scikit-learn.org/stable/modules/classes.html#splitter-classes).
+
+### Dimensionality reduction
+In `s2spy`, users can perform dimensionality reduction on their data. For instance, to perform the Response Guided Dimensionality Reduction (RGDR), the user only needs to configure the RGDR operator and fit it to a precursor field. Then, this cluster can be used to transform the data into the reduced clusters:
+```py
+rgdr = RGDR(eps_km=600, alpha=0.05, min_area_km2=3000**2)
+rgdr.fit(precursor_field, target_timeseries)
+clustered_data = rgdr.transform(precursor_field)
+_ = rgdr.plot_clusters(precursor_field, target_timeseries, lag=1)
+```
+![workflow](./docs/assets/images/rgdr_clusters.png)
+
+Currently, `s2spy` supports [dimensionality reduction approaches](https://scikit-learn.org/stable/modules/classes.html#module-sklearn.cluster) from `scikit-learn`. We are actively working on it to support more approaches in the future.
+
+### Train a model
+More information will follow soon.
+
+### eXplanable AI (XAI) analysis
+More information will follow soon.
+
+## Tutorials
+`s2spy` supports operations that are common in a machine learning pipeline of sub-seasonal to seasonal forecasting research. Tutorials covering supported methods and functionalities are listed in [notebooks](https://github.com/AI4S2S/s2spy/tree/main/notebooks).
```

Comment:
can we add that users need to install `jupyter lab`?

Reply:
Thanks for reminding. It is added now.